www/scripts/codecheckerviewer/ListOfRuns.js

```diff
@@ -71,6 +71,15 @@ function (declare, domConstruct, ItemFileWriteStore, topic, Dialog, Button,
     return container;
   }
 
+  function prettifyStatus(statusCounts) {
+    var stat = [];
```

Comment:
The reference guide (https://www.w3.org/TR/1999/REC-html401-19991224/charset.html#entities) says that it's possible to eliminate but recommend to use it.